### PR TITLE
Add support for 1.20.3

### PR DESCRIPTION
**Pull request by misode**
## Tasks
- [x] Update pack format
- [x] `grass` -> `short_grass`
- [x] Tnt `Fuse` -> `fuse`
- [x] Trident `Trident` -> `trident`
- [x] Remove custom name from some entities
  - All markers, area effect clouds and armor stands: can keep their name
  - ❗Falling Star: name is visible but would cause death messages to no longer mention the cause

### CODE REVIEW

File: gm4_crossbow_cartridges/pack.mcmeta [hunk 1]
```diff
@@ -2,8 +2,12 @@
   "overlays": {
     "entries": [
       {
-        "formats": 18,
+        "formats": [18, 26],
```
[the remark this answers is not in the document]
Comment (BluePsychoRanger) [reply gm4_crossbow_cartridges/pack.mcmeta:5]: Oh you're right. I forgot it was 1.20.1 base with 1.20.2 and 1.20.3 overlays.

**Comment by misode** [comment]
All modules are ready for testing now. A zip with all modules from this PR can be downloaded here: https://github.com/Gamemode4Dev/GM4_Datapacks/suites/18763167905/artifacts/1093072731

**Comment by misode** [comment]
I tested the affected modules in-game, all seem to work (except for #925). There may be a few issues with custom names being visible that are not fixable because we need the name for the death message. An example is falling stars.

**Comment by misode** [comment]
Merging because
1. Some support is better than no support for 1.20.3
2. Having the modules released means free testers
3. I really want this published today

**Pull request merged**